package.json

```diff
@@ -1,8 +1,61 @@
 {
-  "name": "@hostmakerco/example-npm-publish",
-  "version": "1.0.11",
-  "main": "index.js",
-  "repository": "git@github.com:hostmakerco/example-npm-publish.git",
-  "author": "Nikas Praninskas <nikaspran@gmail.com>",
-  "license": "UNLICENSED"
+  "name": "pusher-chatkit-gifted-chat",
+  "version": "1.0.1",
+  "description": "Common components for zendesk",
+  "author": "AlexHayton",
+  "license": "MIT",
+  "repository": "git@github.com:hostmakerco/pusher-chatkit-gifted-chat.git",
+  "main": "dist/pusher-chatkit-gifted-chat.js",
+  "umd:main": "dist/pusher-chatkit-gifted-chat.umd.js",
+  "source": "src/index.ts",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": ">=8",
+    "npm": ">=5"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "dev": "microbundle watch",
+    "start": "REACT_APP_MOCK_ZENDESK=true yarn dev",
```

Comment:
`REACT_APP_MOCK_ZENDESK` is this being used anywhere?

Reply:
Nope. it's going in the bin now :D